[libxlsxwriter] upgraded to 0.8.7

Hi,

I've release a new version of libxlsxwriter, 0.8.7, https://github.com/jmcnamara/libxlsxwriter

If I want to update the version in vcpkg should I submit a pull request? Something like the following?:

```diff
diff --git a/ports/libxlsxwriter/CONTROL b/ports/libxlsxwriter/CONTROL
index 0cf7ed96..8848ce5f 100644
--- a/ports/libxlsxwriter/CONTROL
+++ b/ports/libxlsxwriter/CONTROL
@@ -1,4 +1,4 @@
 Source: libxlsxwriter
-Version: 0.8.6-1
+Version: 0.8.7-1
 Description: Libxlsxwriter is a C library that can be used to write text, numbers, formulas and hyperlinks to multiple worksheets in an Excel 2007+ XLSX file.
 Build-Depends: zlib
diff --git a/ports/libxlsxwriter/portfile.cmake b/ports/libxlsxwriter/portfile.cmake
index f60d2f88..d56b521d 100644
--- a/ports/libxlsxwriter/portfile.cmake
+++ b/ports/libxlsxwriter/portfile.cmake
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO jmcnamara/libxlsxwriter
-    REF RELEASE_0.8.6
-    SHA512 60822dc5f87531edc97cf774e234f25229a605d4430061c24b95c387318e6e05dde1b0f2f433cea14c6f84ee901c1dffe0d174bfd7e2a8459f59bcee900097db
+    REF RELEASE_0.8.7
+    SHA512 20bf09f084808a8db00315848213c550fb809b587ea49ce3b25b310de981c176a44c518452507b6e00ca3f0a8e0056d88a6f575c031d54aa68791575cb9ab285
        HEAD_REF master
        PATCHES
         0001-fix-build-error.patch
```

John

https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/3764/artifacts?artifactName=x64-windows%20port%20build%20failure%20logs&api-version=5.1-preview.5&%24format=zip

`buildtrees\libxlsxwriter\src\ease_0.8.7-5d13031923\include\xlsxwriter\utility.h(20): fatal error C1083: Cannot open include file: 'strings.h': No such file or directory`

--- a/ports/libxlsxwriter/0003-fix-include-file.patch
+++ b/ports/libxlsxwriter/0003-fix-include-file.patch
@@ -1,0 +1,14 @@
+diff --git a/include/xlsxwriter/utility.h b/include/xlsxwriter/utility.h
+index 5b4b365..5f86335 100644
+--- a/include/xlsxwriter/utility.h
++++ b/include/xlsxwriter/utility.h
+@@ -17,7 +17,9 @@
+ #define __LXW_UTILITY_H__
+ 
+ #include <stdint.h>
++#ifndef _MSC_VER
+ #include <strings.h>
++#endif
+ #include "common.h"
+ #include "xmlwriter.h"
+ 

--- a/ports/libxlsxwriter/CONTROL
+++ b/ports/libxlsxwriter/CONTROL
@@ -1,4 +1,4 @@
 Source: libxlsxwriter
-Version: 0.8.6-1
+Version: 0.8.7-1
 Description: Libxlsxwriter is a C library that can be used to write text, numbers, formulas and hyperlinks to multiple worksheets in an Excel 2007+ XLSX file.
 Build-Depends: zlib

--- a/ports/libxlsxwriter/portfile.cmake
+++ b/ports/libxlsxwriter/portfile.cmake
@@ -3,12 +3,13 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH 
     REPO jmcnamara/libxlsxwriter
-    REF RELEASE_0.8.6
-    SHA512 60822dc5f87531edc97cf774e234f25229a605d4430061c24b95c387318e6e05dde1b0f2f433cea14c6f84ee901c1dffe0d174bfd7e2a8459f59bcee900097db
+    REF RELEASE_0.8.7
+    SHA512 20bf09f084808a8db00315848213c550fb809b587ea49ce3b25b310de981c176a44c518452507b6e00ca3f0a8e0056d88a6f575c031d54aa68791575cb9ab285
 	HEAD_REF master
 	PATCHES
         0001-fix-build-error.patch
         0002-fix-uwp-build.patch
+        0003-fix-include-file.patch
 )
 
 if (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")

